opg-php-fpm-1604/docker/my_init.d/40-opg_php_xdebug_remote_host

```diff
@@ -0,0 +1,20 @@
+#!/bin/sh
+
```

Comment:
using 7.0 in the directory name could require this file to be updated for minor version updates. The current version can be determined dynamically or set as a variable in the dockerfile to avoid this

Reply:
@iterative-it combine these rm commands too please